fixtures/js-stubs/span.js

```diff
@@ -0,0 +1,11 @@
+export function Span(params = {}) {
+  return {
+    timestamp: 1657201239.51,
+    start_timestamp: 1657201239.503,
+    op: 'ui.load',
+    span_id: 'a385d9fd52e0c4bc',
+    parent_span_id: 'bdf1a9fae2062311',
+    trace_id: '4d5c2e2102234a7d94102b4f1e41c2bb',
+    ...params,
+  };
+}
```

Comment:
can we move this to `tests/js/sentry-test/performance` so that these test fixtures are contained within the perf team code. we don't have a pattern for top-level fixtures yet so it's better to keep this in one of the perf folders